### ISSUE TITLE
Show relation to oss-guides

### ISSUE THREAD

**Issue by ribose-jeffreylau** [riboseinc/ribose-oss-utils#1]
These two repos go hand-in-hand and should be made clear.

Something like "_Utilities in [`oss-utils`](https://github.com/riboseinc/oss-utils) facilitate implementation of guidelines found in [`oss-guides`](https://github.com/riboseinc/oss-guides)_".

**Comment by skalee** [comment]
I don't understand why they are in separate repositories.

**Comment by ribose-jeffreylau** [comment]
Do you think `oss-utils` would fit inside `oss-guides`?

The original thinking was that to separate "guidelines" and "utilities" as their natures are different.

**Comment by skalee** [comment]
Yes. Given that it includes a single and tiny script, which creates repository according to guidelines, and the whole process is roughly equivalent to copying "for new projects" dir from oss-guides repo, I don't see much reason in keeping these things separate. Also, more complicated utils (e.g. GPG Build Scripts) are kept in their own repositories.

However, if oss-utils is going to be a catalogue of utility script, then it may have sense, perhaps.

**Comment by ribose-jeffreylau** [comment]
Thanks for the feedback!  Yes, this repo will host other utility scripts in the future.  Their scopes will be related to `oss-guides` and quite possibly extend beyond it a bit.  That, however, shouldn't stop the two repos from becoming one if it makes more sense to do so.
